src/components/home/PanelResult.vue

```diff
@@ -18,43 +18,87 @@
       @closing="showDialog = false"
     >
       <template v-slot:title>
-        <span v-if="geneType === 'genesInPanel'">
-          {{ $t('panel-result.dialog.title-genes-in-panel') }}:
-          {{ panelName }}
-        </span>
-        <span v-else>
-          {{ $t('panel-result.dialog.title-genes-not-in-panel') }}:
-          {{ panelName }}
-        </span>
+        {{ $t('panel-result.dialog.title') }}:
+        {{ panelName }}
       </template>
       <template v-slot:content>
-        <v-virtual-scroll :items="genes" height="400" item-height="64">
-          <template v-slot:default="{ item }">
-            <v-list-item :key="item">
-              <v-list-item-content>
-                <v-list-item-title>
-                  <v-chip
-                    :outlined="chipOutlined"
-                    class="ma-1"
-                    :color="formatState(item)"
+        <v-row no-gutters>
+          <v-col
+            cols="6"
+          >
+            <v-simple-table
+              fixed-header
+              height="600px"
+            >
+              <template v-slot:default>
+                <thead>
+                  <tr>
+                    <th class="text-left">
+                      {{ $t('panel-result.dialog.table.genes-found') }}
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr
+                    v-for="item in panelGenes.genesInPanel"
+                    :key="item.gene"
```

Comment:
Make sure the keys are primitives: string or integer. In this instance (and others), item.gene is an object.

Reply:
updated